recipes/pytest-asyncio/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pytest-asyncio" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "ce457148d9a8337679b19742d37f77c5e4eb53a48b626e3ce19464fa1cc25d0f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest >=3.0.2
+
+test:
+    imports:
+      - pytest_asyncio
+
+about:
+  home: http://github.com/pytest-dev/pytest-asyncio
+  license: Apache 2.0
+  #license_file: LICENSE # License file is missing in package
```

Comment:
The issue is fixed upstream in their master branch. We have to wait for their next release to add the license file back.